functions-lab/exercise2/functions2.js

```diff
@@ -0,0 +1,23 @@
+var b1 = document.querySelector('#btn1');
+var b2 = document.querySelector('#btn2');
+var panel = document.querySelector('p');
+
+// b1.addEventListener('click', writeText('Yo'));
+// b2.addEventListener('click', writeText('Nice To Meet You'));
```

Comment:
Note that your solution works but that wasn't the goal of this exercise. I wanted to find a way to modify ONLY the content of the function writeText so the code would still work.

We've already seen in the previous question that we can pass a function object to addEventListener and get the correct result,
But here I insisted on using only a single function (writeText), call it twice and use its return value as the argument.

What should be the return value of writeText so the code would work with my addEventListener calls?

Reply:
thanks, cleaned the code by wrapping it with function so that it will call writeText only when clicked